### PR TITLE
Added Null flag for QRadar Event Correlation Name

### PR DESCRIPTION
**Pull request by JustinJohnWilliams**
## Description
The QUERY passed to QRadar in the `Scripts/script-QRadarGetOffenseCorrelations.yml` excludes offense correlations with a NULL CRE NAME. I added an input flag to include the null values if desired.

### CODE REVIEW

File: Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- contextPath: QRadar.Correlation.CategoryID
+  description: 'The correlation category id '
+- contextPath: QRadar
+  description: QRadar context output
+scripttarget: 0
+dependson:
+  must:
+  - qradar-searches
+tests:
+- No test
```
Comment (DeanArbel) [inline Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations.yml]: Your build currently fails due to:
```
You've failed to provide tests for:
QRadar - Get offense correlations
```
I believe committing this change should fix it
```suggestion
  - No test
```

**Comment by JustinJohnWilliams** [comment]
@DeanArbel looks like that didn't help the tests issue. It appears a few other files have the same configuration as I did before (without the spaces added by your proposed change). Is there maybe another configuration entry I'm missing?

**Comment by DeanArbel** [comment]
You also need to add to `Playbooks/playbook-QRadar_-_Get_offense_correlations_.yml` the `- No test` tag since the playbook is dependent on the script (so it also checks if it has a test).

**Comment by DeanArbel** [comment]
@JustinJohnWilliams Let's clean this branch a bit and merge it into demisto. I will take care of that.

What are the relevant changed files?

**Comment by DeanArbel** [comment]
Closing as there is no change to be found, and author is unresponsive.

**Pull request closed without merging**